hpat/tests/test_series.py

```diff
@@ -72,6 +71,19 @@
     '1234567890',
 ]
 
+def gen_srand_array(size, nchars=8):
+    """Generate array of strings of specified size based on [a-zA-Z] + [0-9]"""
+    accepted_chars = list(string.ascii_letters + string.digits)
+    rands_chars = np.array(accepted_chars, dtype=(np.str_, 1))
+
+    np.random.seed(100)
+    return np.random.choice(rands_chars, size=nchars * size).view((np.str_, nchars))
+
+
+def gen_frand_array(size, min=-100, max=100):
```

Comment:
Please use seed to be able to reproduce the data.

Reply:
I think this is a bad idea to generate input randomly